apps/admin/backend/schema.sql

```diff
@@ -106,6 +106,28 @@ create table ballot_images (
     on delete cascade
 );
 
+create table manual_tallies (
+  id integer primary key,
+  election_id integer not null,
+  precinct_id text not null,
+  ballot_count integer not null,
+  contest_tallies text not null,
+  created_at timestamp not null default current_timestamp,
+  unique (election_id, precinct_id),
+  foreign key (election_id) references elections(id)
+    on delete cascade
+);
+
+create table manual_tally_write_in_candidate_references (
+  manual_tally_id integer not null,
+  write_in_candidate_id varchar(36) not null,
+  primary key (manual_tally_id, write_in_candidate_id),
+  foreign key (manual_tally_id) references manual_tallies(id)
+    on delete cascade,
+  foreign key (write_in_candidate_id) references write_in_candidates(id)
+    on delete cascade
+);
+
```

Comment:
I want to maintain a system, like we have for on-screen adjudication currently, where if you create a write-in candidate it will exist as long as it is referenced. If it's no longer referenced (no more adjudications or no more manual tallies that reference it), it's removed. The alternative is having these records pile up. 

In the future we will have an "official write-in candidates" interface where you can specify official candidates and possibly do any deleting there, but @mattroe is working on the importance / timeline for that feature. Until then, we'll just auto-delete.

The auto-deleting entails some non-trivial overhead (see `store.ts` but it's confined to the store). @kofi-q please let me know if there's any way to make it cleaner or clearer.

Reply:
And @kofi-q another possible idea - instead of managing this reference table, normalize the manual tally data so the links are direct. I thought about this but I don't think that manual tally data normalizes well (we'd have to list metadata and option totals in separate rows with some nullable fields) and it doesn't have other advantages based on how we will query, but I could be convinced.

Reply:
Yeah, in the spirit of "optimise as needed", seems fine to go this route for now. I assume the downside of letting them pile up is that they show up in the UI and make it increasingly congested?

Reply:
Yeah, it's definitely a UI congestion issue rather than a data / performance congestion issue.